service/collector/collector.go

```diff
@@ -46,6 +47,7 @@ type Collector struct {
 	logger    micrologger.Logger
 }
 
+// New creates a new Collector based on a configutation.
```

Comment:
Those comments are also obvious. This also fulfils interface which should be documented already.